src/Illuminate/View/Component.php

```diff
@@ -238,15 +238,10 @@ protected function extractPublicProperties()
             $reflection = new ReflectionClass($this);
 
             static::$propertyCache[$class] = (new Collection($reflection->getProperties(ReflectionProperty::IS_PUBLIC)))
-                ->reject(function (ReflectionProperty $property) {
-                    return $property->isStatic();
-                })
-                ->reject(function (ReflectionProperty $property) {
-                    return $this->shouldIgnore($property->getName());
-                })
-                ->map(function (ReflectionProperty $property) {
-                    return $property->getName();
-                })->all();
+                ->reject(fn (ReflectionProperty $property) => $property->isStatic())
+                ->reject(fn (ReflectionProperty $property) => $this->shouldIgnore($property->getName()))
+                ->map(fn (ReflectionProperty $property) => $property->getName())
```

Comment:
This could use some simplification:
```suggestion
                ->reject
                ->isStatic()
                ->reject(fn (ReflectionProperty $property) => $this->shouldIgnore($property->getName()))
                ->map
                ->getName()
```